src/modal/modal.ts

```diff
@@ -21,7 +21,7 @@ export interface NgbModalOptions {
   /**
    * Size of a new modal window.
    */
-  size?: 'sm' | 'lg';
+  size?: 'sm' | 'lg' | 'md';
```

Comment:
whats the best way to tell someone that there is a "third" size aviable, I would prefer to see the size in the definition, then its "clear" that there is a third size.